js/components/Privacy.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Implements the Privacy rules.
+ *
+ * @providesModule Provacy
+ * @flow
+ */
+
+'use strict'
+
+const React = require('React')
+
+import { StyleSheet, View, Text } from 'react-native'
+
+import { CommonStyles } from './styles'
+
+import I18n from '../i18n'
+
+const styles = StyleSheet.create({
+  content: {
+    backgroundColor: '#D8D8D8',
+    flex: 1,
+    paddingLeft: 24,
+    paddingRight: 24
+  },
+  closeModal: {
+    fontSize: 30,
+    textAlign: 'right',
+    paddingTop: 40,
+    fontFamily: 'Titillium Web',
```

Comment:
@fpersico I'm so much concerned that we're repeating this styles over and over in multiple modules, this duplication will make changing styles in the future MUCH MUCH harder and expensive